app.js

```diff
@@ -198,6 +198,7 @@ app.locals.authProviders = {
   email: config.isEmailEnable,
   allowEmailRegister: config.allowEmailRegister
 }
+app.locals.enableGitlabSnippets = (!config.gitlab.scope || config.gitlab.scope === 'api')
```

Comment:
Can we move the `(!config.gitlab.scope || config.gitlab.scope === 'api')`-part to [`./lib/config/index.js`](https://github.com/hackmdio/codimd/blob/master/lib/config/index.js)?